cmd/esmapping-generator/app/renderer/render.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package renderer
+
+import (
+	"strconv"
+
+	"github.com/jaegertracing/jaeger/cmd/esmapping-generator/app"
+	esTemplate "github.com/jaegertracing/jaeger/pkg/es"
```

Comment:
I think you forgot to change the renamed name to lowercase here.

Reply:
my bad - will fix everywhere, thanks for pointing it out @Ashmita152